frontend/public/module/k8s/index.ts

```diff
@@ -857,6 +857,7 @@ export type K8sKind = {
   verbs?: K8sVerb[];
   shortNames?: string[];
   badge?: BadgeType;
+  className?: string;
```

Comment:
What we need is a color. className is just too board and disconnected from the model itself. Where does the class impl reside? Where is it contributed? Does the className provide a color, background-color, fill, stroke, etc...?
I think it would be better to have a specific color associated with the model.
ResourceLink can apply this color in the style appropriate for the dom node (ie background vs fill). If no color provided, fallback to the existing implementation of classname. We should strive to add colors directly to the models in the future.
cc @spadgett